jovian/utils/colab.py

```diff
@@ -0,0 +1,53 @@
+import sys
+from jovian.utils.api import _h, get_current_user
+from jovian.utils.error import ApiError
+from jovian.utils.logger import log
+from jovian.utils.request import post
+from jovian.utils.shared import _u
+from jovian.utils.request import pretty
+
+
+_colab_file_id = None
+
+
+def in_colab():
+    return 'google.colab' in sys.modules
+
+
+def set_colab_file_id(colab_file_id):
+    global _colab_file_id
+    _colab_file_id = colab_file_id
+
+
+def get_colab_file_id():
+    global _colab_file_id
+    return _colab_file_id
+
+
+def perform_colab_commit(project, privacy):
```

Comment:
TOL: this won't be capturing any hyperparams/metrics

Reply:
Have created a ticket for follow up task